src/index.js

```diff
@@ -1,86 +1,38 @@
-  const checkBoxContainer = document.createElement('div');
-  checkBoxContainer.classList.add('checkbox-container');
-  const label = document.createElement('label');
-  const iconMenu = document.createElement('div');
-  const iconDelete = document.createElement('div');
-  iconMenu.classList.add('menu-container');
-  itemTaskElement.classList.add('list-element');
-  iconMenu.innerHTML = '<i class="fas fa-ellipsis-v"></i>';
-  iconDelete.innerHTML = '<i class="fas fa-trash-alt"></i>';
-  checkBox.type = 'checkbox';
-  checkBox.id = `check${toDoLists.index}`;
-  label.htmlFor = `check${toDoLists.index}`;
-  label.innerText = toDoLists.description;
-
-  label.addEventListener('click', () => {
-    isSelected = !isSelected;
-    if (isSelected === true) {
-      label.style.textDecoration = 'line-through';
-    } else {
-      label.style.textDecoration = 'none';
-    }
-  });
-
-  checkBox.addEventListener('click', () => {
-    if (checkBox.checked) {
-      label.style.textDecoration = 'line-through';
-    } else {
-      label.style.textDecoration = 'none';
-    }
-  });
-
-  checkBoxContainer.append(checkBox, label);
-  itemTaskElement.append(checkBoxContainer, iconMenu);
-  li.appendChild(itemTaskElement);
-  return li;
+if (localStorage.TodoDB) {
+  const localStrg = localStorage.getItem('TodoDB');
+  taskList.tasks = localStrg !== undefined ? JSON.parse(localStrg) : [];
 }
 
-function component() {
-  const container = document.createElement('div');
-  const input = document.createElement('input');
-  const ul = document.createElement('ul');
-  const btnClear = document.createElement('button');
-  container.classList.add('list-container');
-  input.placeholder = 'Add to your list...';
-  btnClear.textContent = 'Clear all completed';
-  btnClear.classList.add('clear-btn');
-  btnClear.disabled = true;
-
-  toDoLists.map((toDoLists) => {
-    ul.appendChild(Todo(toDoLists));
-    return 'Completed';
-  });
-
-  container.append(input, ul, btnClear);
-
-  return container;
-}
-
-const mainContainer = document.querySelector('.main-container');
-
-mainContainer.appendChild(component());
\ No newline at end of file
+const clearAll = document.querySelector('.clear-btn');
+const root = document.querySelector('.main-container');
+const inputElement = document.querySelector('input');
+const ulElement = document.querySelector('ul');
+
+inputElement.addEventListener('change', () => {
+  newTask.description = inputElement.value;
+  newTask.completed = false;
+  newTask.index = taskList.tasks.length;
+  taskList.addTask(new Tasks(newTask.description, newTask.completed, newTask.index));
+});
+
+window.addEventListener('keyup', (e) => {
+  if (e.key === 'Enter') {
+    inputElement.value = '';
+  }
+});
+
+clearAll.addEventListener('click', () => {
+  cleanTasks(taskList);
+  ulElement.innerHTML = '';
+  root.append(taskList.displayTasks(), clearAll);
+});
+
+taskList.displayTasks();
+root.append(clearAll);
```

Comment:
- [ ] as the previous reviewer asked I'm still unable to add a task to the list please make sure to fix that in order to complete the requirements